toolchest_client/tools/tool.py

```diff
@@ -247,7 +248,10 @@ def _postflight(self, output):
                         output_file_path = f"{self.output_path}/{output_name}"
                         sanity_check(output_file_path)
                 else:
-                    sanity_check(self.output_path)
+                    # TODO: verify that this works with absolute pathing
```

Comment:
We should probably add tests for this check

Reply:
Possibly beyond scope of this PR, though

Reply:
I'd agree on both fronts; I'm saving this for the comprehensive pathing PR